libr/crypto/p/crypto_sm4.c

```diff
@@ -0,0 +1,193 @@
+/* *
+ * radare - LGPL - Copyright 2017-2022 - Sylvain Pelissier
```

Comment:
that thing should be in the very first line, not the second. please copy paste the line from another file and update it. the copyright for this code is 2022, because it's new stuff, not 2017. you can add a multiline comment below with the reference url and such but the keep the first line in sync with the rest of r2